build/screener/steps.ts

```diff
@@ -0,0 +1,34 @@
+import * as fs from 'fs'
+import * as Steps from 'screener-runner/src/steps'
+import * as keys from 'screener-runner/src/keys'
+import { reduceSteps, applyThemesBeforeStep } from './utils'
+
+/** converts the arguments into a script passed to Steps.executeScript method
+ * e.g.: steps.executeOnWindow('switchTheme', 'teams')
+ * will call steps.executeScript('window.switchTheme("teams"))
+ */
+Steps.prototype.executeOnWindow = function (methodName: string, ...args: string[]) {
+  const computerArgs = args.length ? `"${args.join('", "')}"` : ''
+  return this.executeScript(`window.${methodName}(${computerArgs})`)
+}
```

Comment:
This method still looks useless, we need it in two cases and the direct usage is much obvious 🤔 I propose to get rid of it for now.

Reply:
will do, we have strange file name conventions... would this be `screener.steps.ts` or `screenerSteps.ts`

Reply:
done